.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
```

Comment:
I'd rather see this more like 90 and 30, but that's really just bikeshedding I guess.

Reply:
my idea is 2 fold:
1) send a remainder that this issue / PR need actions, so the sooner the better
2) when it's really stale close it
Closing is not a death sentence, we can reopen
Also a new comment restart the stale counter, if nobody comment for 60 days I think we can call that stale

Reply:
I'm sold :)